test/cmdlineTests/external_function_pointers_taking_calldata_args/contract.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity >=0.7.0 <0.9.0;
+
+contract C {
+    function d() external {}
+}
```

Comment:
We don't even need the contract here.

Also, please don't hard-code solidity versions in these tests.
```suggestion
pragma solidity *;
```

And, once this test starts working, you might want to add the SPDX comment as well to avoid a warning.